### PR TITLE
Refactor/rename element and header

### PR DESCRIPTION
**Pull request by jost-s**
### Changed

- replaced all occurrences
  - "element" -> "record"
  - "header" -> "action"

### CODE REVIEW

**Comment by jost-s** [comment]
@pdaoust There's mention of "header" and "element" on a couple of the images. Please replace them with updated versions.

- 3.5-commit
- 3.6-action
- 3.7-prev-action-hash
- 4.2-public-entry-commit
- 4.3-private-entry-commit

File: src/glossary.md [hunk 1]
```diff
@@ -557,20 +545,28 @@ A way for agents to share their public keys, prove their authenticity, and revok
...
 
 #### Query arc
 
 A range of [DHT addresses](#dht-address) for which an [agent](#agent) knows a sufficient number of [peers](#peer) who collectively have fault-tolerant coverage (see [saturation](#saturation)) of that range. An agent knows that they can request [DHT data](#dht-data) from anyone within this query arc and get a reasonably authoritative answer (making allowances for [eventual consistency](#eventual-consistency)). This query arc is a looser [neighborhood](#neighborhood) than a [store arc](#store-arc).
 
+#### Record
+
+The data structure that holds an [action](#action) in an [agent](#agent)'s [source chain](#source-chain). Some records are a combination of [action](#action) and [entry](#entry), such as [new-entry actions](#new-entry-action), while others contain all their data inside the record.
```
Comment (maackle) [inline src/glossary.md:564]:
```suggestion
The data structure that holds an [action](#action) in an [agent](#agent)'s [source chain](#source-chain). Some records are a combination of [action](#action) and [entry](#entry), such as [new-entry actions](#new-entry-action), while others contain all their data inside the action.
```

Comment (jost-s) [reply src/glossary.md:564]: @maackle I thought that was wrong too but hesitated. Thanks for the correction!

**Review by maackle — APPROVED** [review]

**Review by pdaoust — COMMENTED** [review]
Looks good to me, with the exception of the images; I've submitted a PR against this PR to change that.

**Pull request merged**